### ISSUE TITLE
Keys stuck when taking screenshot on Mac

### ISSUE THREAD

**Issue by KevinBatdorf** [jaywcjlove/hotkeys#98]
Pressing "command + shift + 4" the command and shift keys get stuck. On the demo it looks like they are only removed when you press them again.

It's reproducible here:
https://wangchujiang.com/hotkeys/

**Comment by jaywcjlove** [comment]
@KevinBatdorf It conflicts with system shortcuts.

**Comment by KevinBatdorf** [comment]
@jaywcjlove Yes, and unfortunately the browser doesn't seem to fire an event when it loses focus to that. 

Maybe the solution would be to allow an option to timeout the multiple keys? As in, `_downKeys` will clear out if no events are fired after 3000ms. It's just a suggestion. I'm not sure what a good solution would be.

**Comment by jaywcjlove** [comment]
@KevinBatdorf I tried a lot of methods and there is no good way to solve them.

**Comment by filipmares** [comment]
Why not just reset the keys when window focus or blurs?

**Comment by kid-icarus** [comment]
I played around with this a bit just now and found some good news. It appears that both Firefox and Safari behave as I would expect. While a user is making a selection and the mouse is down, the user can key up. At that point the keyup event isn't handled, but rather, that keyup event is handled once the mouse is upped.

It appears that chrome is the culprit, and I've [filed a ticket with them](https://bugs.chromium.org/p/chromium/issues/detail?id=1194004): My first thought was that the macOS window manager wasn't correctly dispatching the keyup events to the application, but then I saw that both Firefox and Safari behave as expected. Curious to see how the ticket plays out :)
